### PR TITLE
ADD: Adding new example for working with ARM QC and Xarray Transforma…

### PR DESCRIPTION
**Pull request by AdamTheisen**
…tions.  Updating one example to follow ds convention, and fixing warn message for armfiles

- [x] Documentation reflects changes
- [x] PEP8 Standards or use of linter
- [x] Xarray Dataset or DataArray variable naming follows 'ds' or 'da' naming

### CODE REVIEW

File: examples/workflows/plot_qc_transforms.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+"""
+Transformations and QC
+----------------------
+
+Built-in transformations using xarray do not
+handle ARM quality control appropriately.  This
```
Comment (mgrover1) [inline examples/workflows/plot_qc_transforms.py:6]: I re-phrase as they are not quality-control aware. It is not necessarily something that xarray should inherently do

**Review by mgrover1 — CHANGES_REQUESTED** [review]
One minor comment - otherwise, this looks great!

File: examples/workflows/plot_qc_transforms.py (hunk 1 above)
Comment (AdamTheisen) [reply examples/workflows/plot_qc_transforms.py:6]: Good point!  Will update!

**Comment by AdamTheisen** [comment]
@kenkehoe just wanted to ping you on this example.  Was something like this what you were thinking?

**Review by zssherman — APPROVED** [review]

**Comment by AdamTheisen** [comment]
@mgrover1  does this look good now?

**Review by mgrover1 — APPROVED** [review]
Thanks for making the changes here!

**Comment by kenkehoe** [comment]
@AdamTheisen  overall I think it looks good. I didn't actually run the code so I don't know what the plot looks like. Does the updated QC work?

**Comment by AdamTheisen** [comment]
@kenkehoe the updated qc will not work which is what I try to note in the example and why users should filter their data appropriately ahead of any transformation

**Comment by kenkehoe** [comment]
@AdamTheisen  OK sounds good. Thanks for the example.

**Pull request merged**